portals/publisher/src/main/webapp/source/src/app/components/GlobalPolicies/Policies/SharedComponents/PolicyDropzone.tsx

```diff
@@ -0,0 +1,92 @@
+/*
+* Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+* 
+* WSO2 LLC. licenses this file to you under the Apache License,
+* Version 2.0 (the "License"); you may not use this file except
+* in compliance with the License.
+* You may obtain a copy of the License at
+* 
+* http://www.apache.org/licenses/LICENSE-2.0
+* 
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+import React, { FC, useState } from 'react';
+import { useDrop } from 'react-dnd';
+import PolicyDropzoneShared from 'AppComponents/Shared/PoliciesUI/PolicyDropzone';
+import type { AttachedPolicy, Policy, PolicySpec } from '../Types';
+import AttachedPolicyList from './AttachedPolicyList';
+import PolicyConfiguringDrawer from '../GlobalSpecificComponents/PolicyConfiguringDrawer';
+
+interface PolicyDropzoneProps {
+    policyDisplayStartDirection: string;
+    currentPolicyList: AttachedPolicy[];
+    setCurrentPolicyList: React.Dispatch<
+        React.SetStateAction<AttachedPolicy[]>
+    >;
```

Comment:
Formatting

Reply:
Addressed the required changes by [fddf57f](https://github.com/wso2/apim-apps/pull/515/commits/fddf57fa7308a53094c808e9bf2d2b40ed71b644) and [0ad490c](https://github.com/wso2/apim-apps/pull/515/commits/0ad490c497b08f999babbe521feaf83d029a6698)